tools/mayan-edms/docker-compose.yml

```diff
@@ -21,7 +21,7 @@ x-mayan-container: &mayan-container
 networks:
   psp:
     external: true
-    name: psp-net
```

Comment:
this reminds me, we should really roll this docker-compose into our main one, having two shouldn't be necessary.